include/rocksdb/db.h

```diff
@@ -274,6 +274,17 @@ class DB {
       const std::string& input, std::string* output,
       const CompactionServiceOptionsOverride& override_options);
 
+  // Open DB and trim data to specified timestamp.
+  // The trim_ts specified the user-defined timestamp trim bound.
+  // This API should only be used at a timestamp enabled db instance recovery.
+  // The data with newer timestamp than specified trim bound will be removed
+  // after using this API.
```

Comment:
Since timestamp is a per-cf setting, it's necessary to describe here what happens if some cfs enable timestamp but others do not.

Reply:
Updated comments to add how this API handles column families without timestamps

```c++
// This API should only be used at timestamp enabled column families recovery.
// If some input column families do not support timestamp, nothing will
// be happened to it. The data with timestamp > trim_ts
// will be removed after using this API.
```